client_configuration.go

```diff
@@ -75,10 +84,10 @@ func existsFile(filePath string) (bool, error) {
 func clientConfigPredefinedDirs() []string {
 	homeDir, err := os.UserHomeDir()
 	if err != nil {
-		logger.Warnf("Home dir could not be determined: %w", err)
-		return []string{".", os.TempDir()}
+		logger.Warnf("Unable to access Home directory for Easy Logging configuration search, err: %v", err)
```

Comment:
I think we should change "Easy Logging configuration search" to "client configuration search"

Reply:
Done